glue_jupyter/bqplot/tests/test_bqplot.py

```diff
@@ -305,6 +305,33 @@ def test_imshow_elliptical_brush(app, data_image):
     assert_allclose(roi.yc, 276.75)
 
 
+def test_imshow_circular_annulus_brush(app, data_image):
+
+    v = app.imshow(data=data_image)
+    v.state.aspect = 'auto'
+
+    tool = v.toolbar.tools['bqplot:circannulus']
+    tool.activate()
+    tool.interact.brushing = True
+    tool.interact.selected = [(1.5, 3.5), (300.5, 550)]
+    tool.interact.brushing = False
+
+    roi = data_image.subsets[0].subset_state.roi
+    assert isinstance(roi, CircularAnnulusROI)
+    assert_allclose(roi.xc, 151.00)
+    assert_allclose(roi.yc, 276.75)
+    assert_allclose(roi.outer_radius, 211.375)
+    assert_allclose(roi.inner_radius, 105.6875)
+
+    tool.interact.brushing = True
+    tool.interact.selected = [(150.0, 275.0), (150.0, 325.0)]
+    tool.interact.brushing = False
+    assert_allclose(roi.xc, 151.00)
+    # assert_allclose(roi.yc, 326.75)  # trying to use `move`, but this is not how it works...
```

Comment:
I don't understand this comment. Is there a bug that need fixing here?

Reply:
The "bug" is me not knowing how to work with `tool.interact` to reproduce the interaction of clicking inside the subset, then dragging it 50 pixels upwards and release it (i.e. do a `move` of +50 pixels in y). Just left here as a reminder, but if I don't find a solution, the commented part can be removed again.